pkg/ddevapp/dotddev_assets/providers/pantheon.yaml.example

```diff
@@ -54,6 +54,7 @@ auth_command:
   command: |
     set -eu -o pipefail
     if ! command -v drush >/dev/null ; then echo "Please make sure your project contains drush, ddev composer require drush/drush" && exit 1; fi
+    if ! terminus >/dev/null; then printf "For DDEV, see https://ddev.readthedocs.io/en/stable/users/providers/pantheon/#terminus-and-php-compatibility\n\n" && exit 1; fi
```

Comment:
I added this check to make sure any new pantheon installations show a link to the docs if the command `terminus` fails.